### PR TITLE
Add onboarding doc

### LINKED ISSUE #80: Doc Needed - on and off boarding steering members
Would be good to have a doc for newly elected steering members and those who are retiring/off boarding/[insert leave reason here]. 

what mailing lists do they need to join? other access needed? should they just show up at the next meeting? should they prepare anything (bio for press?, docs to read/policies to become familiar with?)?

### PR DESCRIPTION
**Pull request by nikhita**
Fixes #80 

/hold
/cc @kubernetes/steering-committee

### CODE REVIEW

File: onboarding.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  - [ ] [kubernetes/community]: `sigs.yaml` and auto-generated content
+  - [ ] [kubernetes/org]: `OWNERS_ALIASES` and the `steering-committee`
+    GitHub team
+  - [ ] [kubernetes/funding]: `OWNERS_ALIASES`
+  - [ ] [kubernetes/kubernetes-template-project]: `OWNERS_ALIASES`
+
+- [ ] Add new members with Owner access to the `steering-private`
+  mailing list and remove emeritus members.
+
+- [ ] Grant new members Owner access to the public steering mailing
```
Comment (nikhita) [inline onboarding.md]: I think this is still left for us to do

File: onboarding.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  list and revoke access from emeritus members.
+
+- [ ] Add new members to the `steering-private` slack channel and
+  remove emeritus members.
+
+- [ ] Add new members to the CNCF Service Desk and remove emeritus
+  members.
+
+- [ ] Add new members to the [cncf-kubernetes-maintainers] mailing list.
```
Comment (nikhita) [inline onboarding.md]: Do we want to remove new members too? I got added to the list recently, but from a quick glance it doesn't look like the list includes any confidential info...

File: onboarding.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+- [ ] If a GSuite top-level account is owned by an emeritus member,
```
Comment (nikhita) [inline onboarding.md]: @dims @parispittman should any more details added here? Is the process straight-forward or would it benefit from additional details here?

File: onboarding.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  transfer it to an existing member.
+
+- [ ] If Zoom license credentials were owned by a majority of emeritus
```
Comment (nikhita) [inline onboarding.md]: @parispittman Is `Zoom license credentials` the right term for the zoom creds? :upside_down_face: 

Also, how are the meetings uploaded to youtube today? Is it handled by SC members (akin to SIGs)? Do YouTube creds also need to be shared?

File: onboarding.md (hunk 1 above)
Comment (cblecker) [reply onboarding.md]: Done as a part of https://github.com/kubernetes/k8s.io/pull/419

File: onboarding.md (hunk 3 above)
Comment (parispittman) [reply onboarding.md]: we can link to this - https://github.com/kubernetes/steering#top-level-accounts here so we know where to look quickly for the names of these folks. (i also need to replace jbedas name with mine here hehe)

File: onboarding.md (hunk 4 above)
Comment (parispittman) [reply onboarding.md]: thats right 

we have it automated right now with splain. as long as we hit 'record to the cloud', splain will pick it up and youtube admins will clean it up and post for us within 48 hours.

**Comment by parispittman** [comment]
definitely lgtm! leaving this open for one more set of SC eyes on it in case we are missing something. what about @dims or @timothysc ?

**Comment by dims** [comment]
Do we want to add "setup a private meeting to go over backlog"?

**Comment by nikhita** [comment]
@parispittman @dims Addressed comments.

File: onboarding.md (hunk 2 above)
Comment (dims) [reply onboarding.md]: i don't mind if they want to be around. you don't really need to be on steering to get on this list AFAIK.

**Pull request merged**